### ISSUE TITLE
Build error (TS1323) - target esnext when using dynamic import

### ISSUE THREAD

**Issue by jgoelten** [FabianGosebrink/ASPNETCore-Angular-SignalR-Typescript#18]
Hi @FabianGosebrink 

Thanks for this great sample! I was facing build errors for the frontend using the latest angular 9.x-rc release:

```ts
ERROR in app/app.routing.ts:5:44 - error TS1323: Dynamic import is only supported when '--module' flag is 'commonjs' or 'esNext'.

5   { path: 'dashboard', loadChildren: () => import('./home/home.module').then(m => m.HomeModule) },
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
app/app.routing.ts:6:40 - error TS1323: Dynamic import is only supported when '--module' flag is 'commonjs' or 'esNext'.

6   { path: 'about', loadChildren: () => import('./about/about.module').then(m => m.AboutModule) }
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Changing the module compiler option from '_es2015_' to '_esnext_' in the tsconfig.json file solved the build errors. As described from [https://stackoverflow.com/questions/56375703/angular-8-lazy-loading-modules-error-ts1323-dynamic-import-is-only-supporte](https://stackoverflow.com/questions/56375703/angular-8-lazy-loading-modules-error-ts1323-dynamic-import-is-only-supporte), ts2015 does not support dynamic import.

Am I missing something else or should the module equal to esnext?

jan

**Comment by FabianGosebrink** [comment]
Hey @jgoelten ,

thanks for reporting. Funnily the `tsconfig` is modified on my system but I forgot to commit and push... * doh *. Anyway, thanks for reporting!

Cheers,

Fabian

**Comment by FabianGosebrink** [comment]
Closed by 70dfd40aebd4cff8adcb86f233dac8e035fa5b7d
